arch/arm/src/stm32f7/stm32_qspi.c

```diff
@@ -1133,7 +1133,7 @@ static int qspi0_interrupt(int irq, void *context, FAR void *arg)
   {
     /* Acknowledge interrupt */
 
-    qspi_putreg(&g_qspi0dev, QSPI_FCR_CTCF, STM32_QUADSPI_FCR);
+    qspi_putreg(&g_qspi0dev, QSPI_FCR_CTCF, STM32_QUADSPI_FCR_OFFSET);
```

Comment:
Great finding! There is same issue in STML4 driver. What's impact of the bug?

Reply:
FCR：Flag Clear Register, CTCF: Clear Transfer Complete Flag. 
When transfer a command, it will clear flags first:  https://github.com/YUNEEC/nuttx/blob/d07040d49d16b65b91caad6e37b68b8ccc7226f4/arch/arm/src/stm32f7/stm32_qspi.c#L1954
 then, in function qspi_transmit_blocking(), it will check whether the transfer complete(TCF in status register), then abort QSPI.  
this register offset issue will cause clear flags fail, so, the check TCF operation is always successful and abort QSPI even if the transfer haven't complete. The unprotect flash operation always fail, that no data be followed after write status register 0xA0, is caused by this.

Reply:
But looks we didn't meet such issue?